docker.go

```diff
@@ -535,3 +547,35 @@ to the drive letter:
 	...
 	Z:\ ==> /Z
 `
+
+// volumeExists returns true if the given volume is already created.
+func volumeExists(name string) bool {
```

Comment:
These functions aren't used. Is it expected that they are still there?

Reply:
I forgot to remove them.  docker -v option automatically creates the volume if it doesn't exist.

Removed here: 1491419